singularity-eos/eos/eos_davis.hpp

```diff
@@ -182,7 +186,7 @@ class DavisReactants : public EosBase<DavisReactants> {
 
  private:
   static constexpr Real onethird = 1.0 / 3.0;
-  Real _rho0, _e0, _P0, _T0, _A, _B, _C, _G0, _Z, _alpha, _Cv0;
+  Real _rho0, _spvol, _e0, _P0, _T0, _A, _B, _C, _G0, _Z, _alpha, _i1pa, _Cv0, _A2oB;
```

Comment:
Would it be more helpful if this were `spvol0` or just `v0`?

Reply:
Done